pkg/volume/glusterfs/glusterfs.go

```diff
@@ -243,6 +257,5 @@ func (b *glusterfsBuilder) setUpAtInternal(dir string) error {
 			return nil
 		}
 	}
-	glog.Errorf("Glusterfs: mount failed: %v", errs)
-	return errs
+	return fmt.Errorf("Glusterfs: mount failed: %v", errs)
```

Comment:
nit: should be `glusterfs`, since this is a component name and errors should start lowercase.